### PR TITLE
feat: upgrading oathkeeper-maester version and CRD

### PR DESCRIPTION
**Pull request by LeandroFranciscato**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: helm/charts/oathkeeper-maester/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "v0.1.7"
+appVersion: "v0.1.8"
```
Comment (Demonsthere) [inline helm/charts/oathkeeper-maester/Chart.yaml:2]: This is managed by the CI, please don't modify :)

Comment (Demonsthere) [inline helm/charts/oathkeeper-maester/Chart.yaml:2]:
```suggestion
appVersion: "v0.1.7"
```

**Review by Demonsthere — APPROVED** [review]
🖖

**Comment by LeandroFranciscato** [comment]
So, shouldn't we create a new release to package the `oathkeeper-maester` chart and put it inside of the `oathkeeper` chart?
here: https://github.com/ory/k8s/tree/master/helm/charts/oathkeeper/charts

**Comment by Demonsthere** [comment]
No need, that will be done during the release procedure

**Pull request merged**